supporter-product-data/cloudformation/create-table-stack.sh

```diff
@@ -3,6 +3,6 @@
 # aws cloudformation delete-stack --stack-name supporter-product-data-tables-PROD
 aws cloudformation create-stack \
   --capabilities CAPABILITY_IAM  \
-  --stack-name supporter-product-data-tables-PROD \
+  --stack-name supporter-product-data-tables-CODE \
```

Comment:
Not sure that this script is actually needed but I'm definitely not keen on it pointing to PROD by default